src/game_system.cpp

```diff
@@ -139,13 +139,15 @@ void Game_System::BgmFade(int duration) {
 	force_bgm_play = true;
 }
 
-void Game_System::SePlay(RPG::Sound const& se) {
+void Game_System::SePlay(RPG::Sound const& se, bool stop_sounds) {
 	static bool ineluki_warning_shown = false;
 
 	if (se.name.empty()) {
 		return;
 	} else if (se.name == "(OFF)") {
-		Audio().SE_Stop();
+		if (stop_sounds) {
```

Comment:
This one could be moved upwards, e.g.: `if ((se.name == "(OFF)" && stop_sounds) {`